nemo/collections/nlp/models/token_classification/punctuation_capitalization_config.py

```diff
@@ -212,6 +213,49 @@ class PunctuationCapitalizationModelConfig:
     documentation and `primer <https://github.com/NVIDIA/NeMo/blob/main/tutorials/00_NeMo_Primer.ipynb>_ tutorial."""
 
 
+@dataclass
+class PunctuationCapitalizationLexicalAudioModelConfig(PunctuationCapitalizationModelConfig):
+    train_ds: Optional[PunctuationCapitalizationTrainDataConfig] = None
+    """A configuration for creating training dataset and data loader."""
+
+    validation_ds: Optional[PunctuationCapitalizationEvalDataConfig] = None
+    """A configuration for creating validation datasets and data loaders."""
+
+    test_ds: Optional[PunctuationCapitalizationEvalDataConfig] = None
+    """A configuration for creating test datasets and data loaders."""
+
+    pretrained_audio_encoder: str = MISSING
+    """A configuration for restoring pretrained audio encoder"""
+
+    freeze_audio_encoder: bool = False
+    """Freeze audio encoder weight and add LSTM module on top of it"""
+
+    fusion_inner_size: int = 2048
+    """Fusion inner size"""
+
+    fusion_num_attention_heads: int = 4
+    """Number of attention heads to use in fusion"""
+
+    fusion_num_layers: int = 4
+    """"Number of layers to use in fusion"""
+
+    restore_lexical_encoder_from: Optional[str] = None
+    """"Path to .nemo checkpoint to load weights from"""
+
+    use_adapters: Optional[bool] = False
+    """use adapters for audio encoder"""
+
+    adapter_config: Optional[LinearAdapterConfig] = None
+
+    use_weighted_loss: Optional[bool] = False
```

Comment:
could you remove redundant empty lines to improve the readability so that all related args are grouped together with a comment above the group? e.g., group `use_adapters` and `adapter_config`